docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3'
+services:
+  jenkins:
+    image: jenkins/jenkins:lts
```

Comment:
It is always a good practice to pin the version of image being used instead of using `latest`, `lts` tags

Reply:
I was initially considering using a specific version tag, but then I realized it would be better to use LTS instead. By using LTS, the user won't need to update the version tag frequently.